pkg/rancher-desktop/backend/containerClient/nerdctlClient.ts

```diff
@@ -134,6 +134,26 @@ export class NerdctlClient implements ContainerEngineClient {
     }
   }
 
+  async waitForReady(): Promise<void> {
```

Comment:
I was expecting this function for nerdctl to have the same structure as the one in `pkg/rancher-desktop/backend/containerClient/mobyClient.ts`, although checking two commands instead of one. Probably could use a comment explaining why you don't need to do 10 successful checks before accepting it, like you do with moby (or it's a bug here).

Reply:
`mobyClient` needs the retry because (on Windows) the proxy process dies and restarts multiple times (causing flapping).  Here we're waiting on the `containerd`/`buildkitd` processes on the VM, which once they start working do not die by themselves.

If anything, the fix should be to change the proxy process to stop flapping. That would involve fixing #4551 instead.